src/css/mg_line_brushing.css

```diff
@@ -0,0 +1,19 @@
+.mg-brush-container {
+  cursor: crosshair; }
+
+.mg-brush-container.mg-brushing {
+  cursor: ew-resize; }
+
+.mg-brushed, .mg-brushed * {
+  cursor: zoom-out !important; }
+
+.mg-brush rect.mg-extent {
+  fill: rgba(0, 0, 0, 0.3); }
+
+.mg-brushing-in-progress {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none; }
```

Comment:
These lines can go straight into `dist/metricsgraphics.css` (please use the style there)